README.md

```diff
@@ -44,39 +74,18 @@ acquisition, and analysis.
 
    - Instrutech/HEKA ITC:
      * [16](http://www.heka.com/downloads/hardware/manual/itc16.pdf)
-      * [18](http://www.heka.com/downloads/hardware/manual/m_itc18.pdf)
+     * [18](http://www.heka.com/downloads/hardware/manual/m_itc18.pdf)
      * [1600](http://www.heka.com/downloads/hardware/manual/m_itc1600.pdf)
 
-     ITC devices are at the End-Of-Service-Life (EOSL). National Instruments DACs are recommended for MIES users looking
-for new hardware.
+     ITC devices are at the End-Of-Service-Life (EOSL). National Instruments DACs are recommended for MIES users looking for new hardware.
 
 - Amplifier: Molecular Devices [700B](https://www.moleculardevices.com/products/axon-patch-clamp-system/amplifiers/axon-instruments-patch-clamp-amplifiers)
 - Pressure control (optional): ITC 18/1600 or National Instruments [USB 6001](https://www.ni.com/en-us/support/model.usb-6001.html)
 
```

Comment:
Please add the pressure regulator company name, product link, part name, and part number.
Product link: https://proportionair.com/product/qpv/
Company name: Proportion Air
Part name: QPV regulator (custom, see part No), -10 psi to +10 psi
Part no. QPV1TBNEEN10P10PSGAXLDD